readme.md

```diff
@@ -14,6 +14,26 @@ This stores your local credentials
 
 yarn get-tests
 
+# Setup container api for testing with clients
+
+## Podman
+
+As your user, run podman to serve the api:
+```
+podman system service -t 0 /tmp/podman.sock
```

Comment:
oh and one more thing, this command did not work and failed with this err: 
```
Error: API Service endpoint scheme "" is not supported. Try tcp:///tmp/podman.sock
 or unix:///tmp/podman.sock
 ```
 
 so I had to change it to this/specify the API scheme as `unix`: `podman system service -t 0 unix:///tmp/podman.sock`